### PR TITLE
Add page for individual podcast episodes

### PR DESCRIPTION
**Pull request by beyang**
Adds a page for individual podcast episodes:

image

### CODE REVIEW

**Comment by uwedeportivo** [comment]
looks very nice

File: website/src/pages/podcast.tsx [hunk 1]
```diff
@@ -2,6 +2,51 @@ import * as React from 'react'
...
+interface HTMLParts {
+    guestsHTML: string
+    audioHTML: string
+    summaryHTML?: string
+    transcriptHTML?: string
+    showNotesHTML?: string
+}
+
+export function getHTMLParts(html: string): HTMLParts {
+    const doc = document.createElement('html')
```
Comment (aileenrose) [inline website/src/pages/podcast.tsx:39]:
```suggestion
    const doc = document.createElement('html')
```
I tried to review locally and received an error for 'document'.  Should this use:
public componentDidMount() ?

Comment (beyang) [reply website/src/pages/podcast.tsx]: I fixed this issue, so it should work locally now. Still need to fix the outstanding build errors.

**Comment by bobheadxi** [comment]
Excited to see this! A bit of an aside: the Sourcegraph logo could probably use some whitespace around it, since right now it looks a bit strange against dark backgrounds going all the way to the edge:

image

(our github org logo has a similar issue on dark backgrounds, ie the GitHub mobile app)

**Pull request merged**